airbyte-webapp/.eslintrc.js

```diff
@@ -88,6 +88,7 @@ module.exports = {
     "react/jsx-fragments": "warn",
     "react/jsx-no-useless-fragment": ["warn", { allowExpressions: true }],
     "react/self-closing-comp": "warn",
+    "react/style-prop-object": ["warn", { allow: ["FormattedNumber"] }],
```

Comment:
:information_source: Needed so that eslint doesn't complain anymore that `style` needs to be an object on `FormattedNumber` (where it doesn't need to be an object actually).